sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSource.cs

```diff
@@ -22,15 +22,15 @@ internal DataFeedSource(DataFeedSourceType dataFeedSourceType)
             dataFeedDetail switch
             {
                 AzureApplicationInsightsDataFeed d => new AzureApplicationInsightsDataFeedSource(d.DataSourceParameter),
-                AzureBlobDataFeed d => new AzureBlobDataFeedSource(d.DataSourceParameter),
+                AzureBlobDataFeed d => new AzureBlobDataFeedSource(d.DataSourceParameter, d.AuthenticationType),
```

Comment:
This is the transform service => public. We're now passing the authentication type / credential id returned by the service to some data sources.